src/components/Inputs/WrappedColorPicker.tsx

```diff
@@ -0,0 +1,116 @@
+import { Button, ColorPicker, ColorPickerProps } from "antd";
```

Comment:
Convenience component that wraps Ant's `ColorPicker` component and fixes several accessibility issues with it. The change was motivated by me needing to assign an `id` to a `ColorPicker` for labeling, which Ant doesn't support without changing its triggering button component.

<img width="337" height="393" alt="Image" src="https://github.com/user-attachments/assets/cc73c673-803b-4dd2-8354-42d6c383a09f" />